dsgrid/dataset/dataset_schema_handler_one_table.py

```diff
@@ -50,9 +48,24 @@ def _check_one_table_data_consistency(self):
         logger.info("Check one table dataset consistency.")
         dimension_types = set()
         pivoted_cols = set()
+        dimension_type_to_column = {}
+        column_to_dimension_type = {}
 
         time_dim = self._config.get_dimension(DimensionType.TIME)
-        time_columns = time_dim.get_timestamp_load_data_columns()
+        if self._config.model.data_schema.column_type == ColumnType.DIMENSION_TYPES:
+            time_columns = time_dim.get_timestamp_load_data_columns()
+            for dimension_type in DimensionType:
+                dimension_type_to_column[dimension_type] = dimension_type.value
+                column_to_dimension_type[dimension_type.value] = dimension_type
```

Comment:
My solution completes assignment of the dictionaries in one iteration. I know that doesn't really matter because the size is tiny. Is this really harder to read?